.github/workflows/master.yaml

```diff
@@ -17,19 +17,19 @@ jobs:
       fail-fast: false
     steps:
       - name: Setup ROS 2
-        uses: ros-tooling/setup-ros@0.1.1
+        uses: ros-tooling/setup-ros@v0.2
```

Comment:
Thanks for updating this. Maybe 0.2.1 is better, right?

Reply:
seems 0.2.1 is not exist... ref:https://github.com/paeunt/cascade_lifecycle/runs/2735240985

Reply:
As showed in https://github.com/ros-tooling/action-ros-ci/releases, there is no v0.2.1, only v0.2 for it. I'll change it back.